doc/tutorials/contributing_code.rst

```diff
@@ -0,0 +1,268 @@
+
+  JPEG2000MultiComponentLossless
+  JPEG2000MultiComponent
+  RLELossless
+  NewDefinedUID
+
+When making changes, and especially when adding new features, its important
+that they're documented. It's very difficult for users to find and
+understand how to use code that hasn't been documented, or whose documentation
+contains errors. For more information on how to properly document *pydicom*
+see :doc:`writing documentation</guides/writing_documentation>`.
+
+Now we run the tests again so we can see whether or not the code we added is
+working::
+
+  $ pytest test_uid.py
+
+Everything should pass. If it doesn't, make sure you've correctly added the
+new UID. Once you're happy that the tests in ``test_uid.py`` are working you
+should make sure the entire test suite passes::
+
+  $ pytest
+
+
+Preview your changes
+====================
+It's a good idea to go through all the changes you've made by first staging
+and then displaying the difference between the current copy and the initial
+version we first checked out with::
+
+  $ git add --all
+  $ git diff --cached
+
+You can scroll through the output using the up and down keys and quit with
+**q**. Lines with a **-** in front will be removed and lines with a **+**
+added. If everything looks good then its time to commit the changes.
+
+
+Commit your changes and make a pull request
+===========================================
+To commit the changes::
+
+  $ git commit
+
+This will open a text editor so you can add the commit message. Alternatively,
+if you only want a short commit message you can do::
+
+  $ git commit -m "Add NewDefinedUID"
+
+Which will commit with the message *"Add NewDefinedUID"*. After committing the
+patch, send it to your fork::
+
+  $ git push origin new-uid
+
+You can create a pull request by visiting the `pydicom GitHub page
+<https://github.com/pydicom/pydicom>`_ where you
+should see your branch under *"Your recently push branches"*. Click *"Compare &
+pull request"* and fill out the title (with a ``[WIP]`` prefix, i.e.
+``[WIP] Add NewDefinedUID to uid.py``) and follow the  instructions in the
+main entry window.
+
+To submit the pull request (PR) for real - **please don't do this for
+this example!** - then on the next page you would click *"Create pull
+request"*.
+Creating the PR would automatically start our checks; that the tests pass and
+the test coverage is good, that the documentation builds OK, etc.
+
+If all the checks passed and you were happy with your changes, you'd change
+the PR title prefix to ``[MRG]``. This would indicate that you considered the
+PR ready to be reviewed and merged into the main branch. You could also ask
+for a review or help at any point after creating the PR.
+
+What happens next?
+==================
+One or more reviewers would look at your pull-request and may make suggestions,
+ask for clarification or request changes. Once the reviewers were happy then
+the pull request would be approved and your changes merged into the
```

Comment:
One the reviewers were (or are?) happy, the pull request ...

Reply:
Its a hypothetical review, since I told them not to do an actual PR. Using "are" implies that its not hypothetical.

Reply:
Yeah, I got that, I wasn't sure if it also possible... obviously not. Edited the comment.